access/pagerduty/app.go

```diff
@@ -235,6 +239,25 @@ func (a *App) onWatcherEvent(ctx context.Context, event types.Event) error {
 	}
 }
 
+// maybeProcessRequest checks request against the configured filter and decides if request should be processed.
+// When Approve field is unset or contains "*", all requests will be processed.
+// When Approve field is configured, requests with roles specified in config will be processed.
```

Comment:
Can you please re-word this comment?
I don't think we have an `Approve` field
We are also missing the field name we use to filter out the requests (in this case: roles)

Reply:
It should never be unset/empty according to the `CheckAndSet` method